lib/location/method_cellular.c

```diff
@@ -103,14 +103,19 @@ static int method_cellular_ncellmeas_start(void)
 
 static void method_cellular_positioning_work_fn(struct k_work *work)
 {
+	struct multicell_location_params params = { 0 };
 	struct multicell_location location;
 	struct location_data location_result = { 0 };
+	int64_t ncellmeas_start_time;
+	int64_t ncellmeas_time;
 	int ret;
 	struct method_cellular_positioning_work_args *work_data =
 		CONTAINER_OF(work, struct method_cellular_positioning_work_args, work_item);
 	const struct location_cellular_config cellular_config = work_data->cellular_config;
 
-	location_core_timer_start(cellular_config.timeout);
+	location_core_timer_start(cellular_config.timeout + 5);
```

Comment:
There could be a comment for `+ 5`.

Reply:
This is a leftover from testing so will remove this.